aws-reactive/.sts4-cache/classpath-data.json

```diff
@@ -0,0 +1 @@
Philippe/.m2/repository/net/bytebuddy/byte-buddy-agent/1.10.2/byte-buddy-agent-1.10.2-sources.jar","javadocContainerUrl":"file:/C:/Users/Philippe/.m2/repository/net/bytebuddy/byte-buddy-agent/1.10.2/byte-buddy-agent-1.10.2-javadoc.jar","isSystem":false,"isOwn":false,"isTest":false,"isJavaContent":false},{"kind":"binary","path":"C:\\Users\\Philippe\\.m2\\repository\\org\\objenesis\\objenesis\\2.6\\objenesis-2.6.jar","sourceContainerUrl":"file:/C:/Users/Philippe/.m2/repository/org/objenesis/objenesis/2.6/objenesis-2.6-sources.jar","javadocContainerUrl":"file:/C:/Users/Philippe/.m2/repository/org/objenesis/objenesis/2.6/objenesis-2.6-javadoc.jar","isSystem":false,"isOwn":false,"isTest":false,"isJavaContent":false},{"kind":"binary","path":"C:\\Users\\Philippe\\.m2\\repository\\org\\apache\\maven\\surefire\\surefire-logger-api\\2.21.0\\surefire-logger-api-2.21.0.jar","sourceContainerUrl":"file:/C:/Users/Philippe/.m2/repository/org/apache/maven/surefire/surefire-logger-api/2.21.0/surefire-logger-api-2.21.0-sources.jar","javadocContainerUrl":"file:/C:/Users/Philippe/.m2/repository/org/apache/maven/surefire/surefire-logger-api/2.21.0/surefire-logger-api-2.21.0-javadoc.jar","isSystem":false,"isOwn":false,"isTest":false,"isJavaContent":false},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\main\\java","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\classes","javadocContainerUrl":"file:/C:/Users/Philippe/work/baeldung/repo/tutorials/aws-reactive/target/site/apidocs","isSystem":false,"isOwn":true,"isTest":false,"isJavaContent":true},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\test\\java","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\test-classes","javadocContainerUrl":"file:/C:/Users/Philippe/work/baeldung/repo/tutorials/aws-reactive/target/site/apidocs","isSystem":false,"isOwn":true,"isTest":true,"isJavaContent":true},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\main\\resources","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\classes","isSystem":false,"isOwn":true,"isTest":false,"isJavaContent":false},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\test\\resources","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\test-classes","isSystem":false,"isOwn":true,"isTest":true,"isJavaContent":false},{"kind":"source","path":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\src\\test\\java","outputFolder":"C:\\Users\\Philippe\\work\\baeldung\\repo\\tutorials\\aws-reactive\\target\\test-classes","javadocContainerUrl":"file:/C:/Users/Philippe/work/baeldung/repo/tutorials/aws-reactive/target/site/apidocs","isSystem":false,"isOwn":true,"isTest":false,"isJavaContent":false}]}
```

Comment:
Don't think we need this file in the PR.

Reply:
Removed